hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import styles from "./CartCard.module.scss";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import {
+    Card,
+    CardMedia,
+    Select,
+    FormControl,
+    MenuItem,
+    IconButton,
+    Paper,
+    InputLabel,
+} from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+
+const Selections = (currentStock) => {
+    const items = [];
+    let i = 0;
+
+    for (i = 1; i < currentStock + 1; i++) {
+        items.push(
+            <MenuItem key={i} role="quantity" value={i.toString()}>
+                {i}
+            </MenuItem>
+        );
+    }
+    return items;
+};
+
+export const CartCard = ({
+    image,
+    title,
+    currentStock,
+    closeChange,
+    handleChange,
+    item,
+}) => {
+    return (
+        <Grid className={styles.Cart} item>
+            <Paper className={styles.Cart} elevation={0}>
+                <CardMedia
+                    className={styles.CartPic}
+                    image={image}
+                    alt={title}
+                    component="img"
+                />
+            </Paper>
+            <Card className={styles.CartParts} elevation={0}>
+                <Typography variant="body2" className={styles.CartName}>
```

Comment:
If you look on the Figma frame 7.1 - Cart Error, if there's an error, there's a message in red under the title that shows up. Have a prop called `isError` for now to handle it